poppy/provider/akamai/background_jobs/check_cert_status_and_update/check_cert_status_and_update_tasks.py

```diff
@@ -28,6 +28,90 @@
+        service_controller, self.providers = \
+            memoized_controllers.task_controllers('poppy', 'providers')
+        self.akamai_driver = self.providers['akamai'].obj
+        self.sc = self.akamai_driver.service_controller
+        self.akamai_conf = self.akamai_driver.akamai_conf
+
+        self.existing_hosts = []
+
+    def execute(self, property_spec):
+        """Get latest Akamai property, get its hostnames"""
+        self.property_id = self.akamai_driver.papi_property_id(property_spec)
+
+        LOG.info('Starting to get latest version for property: %s'
+                 % self.property_id)
+        resp = self.akamai_driver.akamai_papi_api_client.get(
+            self.akamai_driver.akamai_papi_api_base_url.format(
+                middle_part='properties/%s' % self.property_id)
+        )
+        if resp.status_code != 200:
+            raise RuntimeError('PAPI API request failed.'
+                               'Exception: %s' % resp.text)
+        else:
+            a_property = json.loads(resp.text)['properties']['items'][0]
+            latestVersion = a_property['latestVersion'] or 0
+            production_version = a_property['productionVersion'] or -1
+            staging_version = a_property['stagingVersion'] or -1
+
+            max_version = max(latestVersion, production_version,
+                              staging_version)
+
+            LOG.info('Starting to get details for latest property'
+                     ' version: %s' % max_version)
+
+            resp = self.akamai_driver.akamai_papi_api_client.get(
+                self.akamai_driver.akamai_papi_api_base_url.format(
+                    middle_part='properties/%s/versions/%s' % (
+                        self.property_id, str(max_version)))
+            )
+            if resp.status_code != 200:
+                raise RuntimeError('PAPI API request failed.'
+                                   'Exception: %s' % resp.text)
+
+            # if the latest version is pending state, stop execution
+            production_status = \
+                json.loads(resp.text)['versions']['items'][0][
+                    'productionStatus']
+
+            if production_status.upper() == 'PENDING':
+                err_message = (
+                    'Aborting task/flow execution.'
+                    'Akamai property version {0} is in {1} status.'.format(
+                        max_version,
+                        production_status
+                    )
+                )
+                LOG.error(err_message)
+                raise ValueError(err_message)
+
+        LOG.info("Starting to get property hostnames.")
+
+        resp = self.akamai_driver.akamai_papi_api_client.get(
+            self.akamai_driver.akamai_papi_api_base_url.format(
+                middle_part='properties/%s/versions/%s/hostnames' %
+                (self.property_id,
+                 str(max_version)))
+        )
+        if resp.status_code != 200:
```

Comment:
is it fine if we change "if resp.status_code != 200:" to "if not resp.status_code.ok:"